src/structure/pure/pagejump.js

```diff
@@ -48,7 +47,15 @@ design.basicdesign.addCreator('pagejump', {
       'align'   : $e.attr('align'),
       'target'  : target
     } );
-    return ret_val + actor + '</div>';
+    var info = '';
+    var infoWidget = $('infowidget > *', $e).first()[0];
```

Comment:
it is not dependant from the infoaction plugin, but uses the info-widget, which is, like the pagejump itself, part of the pure-structure. IMHO this is no bad architecture, because they "live in the same namespace". The other solution would be to copy the whole code from info.js to pagejump.js, and that would be bad architecture/coding e.g. redundant code.